website/src/pages/forum.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Layout from '@theme/Layout';
+import Head from '@docusaurus/Head';
+import { DiscourseFeed } from '@site/src/components/discourse';
+
+function Events() {
+  return (
+    <Layout>
+      <Head>
+        <title>dbt Discourse</title>
+        <meta name="description" content="Description here" />
```

Comment:
@JKarlavige How about "Recent interesting discussions from across the dbt Forum" for now